packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentMethods.js

```diff
@@ -6,15 +6,10 @@ import { usePaymentMethods } from '@magento/peregrine/lib/talons/CheckoutPage/Pa
 
 import { mergeClasses } from '../../../classify';
 import Radio from '../../RadioGroup/radio';
-import CreditCard from './creditCard';
 import paymentMethodOperations from './paymentMethods.gql';
 import defaultClasses from './paymentMethods.css';
 
-const PAYMENT_METHOD_COMPONENTS_BY_CODE = {
-    braintree: CreditCard
-    // checkmo: CheckMo,
-    // etc
-};
+import payments from './paymentMethodByCode.js';
```

Comment:
How about `import paymentMethods from paymentMethodMapper`? We can omit `.js` in the import.